{"level":"error","message":"Sample error! Sample error!","stack":"Error: Sample error!\n    at /Users/manu/Projects/lbs/routes/genres.js:16:15\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/manu/Projects/lbs/middleware/async.js:5:5"}
{"level":"error","message":"Sample error! Sample error!","stack":"Error: Sample error!\n    at /Users/manu/Projects/lbs/routes/genres.js:16:15\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/manu/Projects/lbs/middleware/async.js:5:5"}
{"level":"error","message":"Sample error! Sample error!","stack":"Error: Sample error!\n    at /Users/manu/Projects/lbs/routes/genres.js:16:15\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /Users/manu/Projects/lbs/middleware/async.js:5:5"}
{"level":"error","message":"Something failing badly Something failing badly","stack":"Error: Something failing badly\n    at Object.<anonymous> (/Users/manu/Projects/lbs/index.js:45:7)\n    at Module._compile (node:internal/modules/cjs/loader:1126:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)\n    at Module.load (node:internal/modules/cjs/loader:1004:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:839:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)\n    at node:internal/main/run_main_module:17:47"}
